README.md

```diff
@@ -49,7 +57,7 @@ where _interval_ is the interval indicating how frequently the statistics is col
 
 ## Example Plots
 
-The script plots different charts from SRT statistics. For example, a chart to visualize statistics on the packets being sent, lost, retransmitted, dropped or on flight:
+The script plots different charts from SRT statistics, an example .csv file can be found in the examples folder, together with the corresponding output html. . For example, a chart to visualize statistics on the packets being sent, lost, retransmitted, dropped or on flight:
```

Comment:
Suggested change:
"The script plots different charts from SRT statistics, an example .csv file can be found in the examples folder, together with the corresponding output html. ." -> 
"The script plots different charts from SRT statistics. Examples of `.csv` files and corresponding `.html` output can be found in the [examples](./examples) folder."

Reply:
I would suggest to create a folder "srt-live-transmit" under "examples" folder and put your files in there. It would be also helpful if you could generate sender side statistics and add examples for it as well.

Reply:
resolved in https://github.com/koenkarsten/srt-stats-plotting/commit/718676727c4f42878ea9030a123704e6ee2efe63